server/api/course/course.model.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var mongoose = require('bluebird').promisifyAll(require('mongoose'));
+var Schema = mongoose.Schema;
+
+var CourseSchema = new Schema({
+  name: String,
+  description: String,
+  students: [{type : Schema.Types.ObjectId, ref: 'User'}],
+  active: Boolean
```

Comment:
@leej42 I think that's a good idea- but wasn't in the original spec- so i'm going to recommend we go sort of MVP style